api-gateway/composing-many-apis/build.gradle.kts

```diff
@@ -40,7 +40,7 @@ allprojects {
 }
 
 tasks.named("build") {
-    dependsOn("runTest")
+    dependsOn("runFunctionalTest")
```

Comment:
I think the original version is valid as far as you use a released starter kit version:
https://github.com/Knotx/knotx-starter-kit/archive/2.0.0.zip

Please check ZIPed gradle scripts.

Reply:
fixed, in starterkit runTest is used